src/sm/keycards.asm

```diff
@@ -121,7 +121,11 @@ keycard_update_layer3:
     dex #2
     bpl -
 
+    LDA $80FF52 ; load config bits
```

Comment:
I'm not sure if we should be using the seed config field for this. That field has very limited amount of data and can not properly represent all the configuration that might be needed for things. I think the previous solution with a "config_keysanity" is better here, or a specific flag for these keycard markers.